docs/tutorials/standalone/Soil/freezing_front.jl

```diff
@@ -109,16 +108,16 @@ hcm = vanGenuchten{FT}(; α = vg_α, n = vg_n);
 ν_ss_om = FT(0.3)
 ν_ss_quartz = FT(0.7)
 ν_ss_gravel = FT(0.0)
-params = Soil.EnergyHydrologyParameters{FT}(;
-    ν = ν,
-    ν_ss_om = ν_ss_om,
-    ν_ss_quartz = ν_ss_quartz,
-    ν_ss_gravel = ν_ss_gravel,
-    hydrology_cm = hcm,
-    K_sat = K_sat,
-    S_s = S_s,
-    θ_r = θ_r,
-    earth_param_set = earth_param_set,
+params = Soil.EnergyHydrologyParameters(
+    FT;
+    ν,
```

Comment:
I wonder if it would be better to continue using the keyword arguments (e.g. passing `\nu = \nu`) so that we don't rely on the positional order of the arguments. I'm okay with either way, just a thought!

Reply:
I think that the semicolon after the FT is enough to ensure that they are read as kwargs if the variable name matches the keyword:

```julia
f(a;b = 2) = a*b
f(2;b = 3)
b = 4
f(2; b)
```

Reply:
ahh I didn't realize! nice